test_src/SerializationTest.cpp

```diff
@@ -56,6 +57,26 @@ namespace voltdb {
 
 static const double EPSILON = 1.0e-14;
 
+void dumpBytes(char *bytes, size_t length) {
+    for (int i = 0; i < length; ++i) {
+        printf("%02x ", bytes[i] && 0xff);
+        if (i % 16 == 0) {
+            printf("\n");
+        }
+    }
+    printf("\n");
+}
+
+
+void compareBuffers(char *first, char*second, size_t length) {
+    for (int i = 0; i < length; ++i) {
+        if (first[i] != second[i]) {
+            printf("%d (%d %d) ", i, (first[i] & 0xff), (second[i] & 0xff));
```

Comment:
I had in hex earlier thn changed it to dec when was debugging an issue. I will change it to hex as for general purpose hex will be better.